vm/devices/firmware/firmware_uefi/src/service/nvram/mod.rs

```diff
@@ -348,10 +348,16 @@ impl NvramServices {
             use uefi_specs::uefi::nvram::vars::SECURE_BOOT;
 
             let (vendor, name) = SECURE_BOOT();
-            let attr = EfiVariableAttributes::DEFAULT_ATTRIBUTES_VOLATILE;
+            // Older versions of OpenHCL (and Hyper-V, closed-source HCL, etc. ) may have created
+            // SecureBoot variable with NV attribute, which doesn't match the UEFI spec.  If this
+            // variable already exists, use its existing attributes.
```

Comment:
Is this really the behavior we want? Why not delete the current variable, and re-inject it with the correct attributes?

Reply:
That may be a viable alternative.  

Initially I discarded that approach in case some other component (including the original HCL) might break if the attributes changed.  But I think maybe it's no different than if OpenHCL creates the variable store first.